app/views/measures/grouped/_tariff_duty_calculator_link.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-inset-text govuk-inset-text--s feature-panel--shaded explainer">
```

Comment:
Don't forget to nest these in

```erb
<% if declarable.calculate_duties? %>
...
<% end %>
```

Reply:
maybe this is would be redundant,
this view is called on this conditional:

`<% if xi_import_measures.customs_duties.present? %> `